### PR TITLE
Update ibm-cloud-sdk-core to 2.4.5

### PR DESCRIPTION
**Pull request by emlaver**
## PR summary

Fixes: #3 

**Note: An existing issue is [required](https://github.com/IBM/cloudant-node-sdk/blob/master/CONTRIBUTING.md#PRs) before opening a PR.**

## PR Checklist

Please make sure that your PR fulfills the following requirements:

- [x] The commit message follows the
[Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [x] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## What is the current behavior?
Current version of `ibm-cloud-sdk-core` is `2.4.0`

## What is the new behavior?
`ibm-cloud-sdk-core` is updated to `2.4.5`

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by vmatyusGitHub** [comment]
The CI has some problem.

File: package.json [hunk 1]
```diff
@@ -38,9 +38,10 @@
     "node": ">=10.0.0"
   },
   "dependencies": {
+    "@types/extend": "^3.0.1",
```
Comment (ricellis) [inline package.json:41]: I don't think we should add this dependency it is not required - I think the problem is basically that we need to run the latest generator to get source code that doesn't require this type (see e.g. https://github.com/IBM/platform-services-node-sdk/pull/40).

I think instead we could merge the `2.4.5` upgrade commit alongside the latest generator source (i.e. close this PR and add the core change commit onto that branch when it is ready).

**Review by ricellis — CHANGES_REQUESTED** [review]
I think we should just hold this PR until the 3.12 based generated source update is ready.

File: package.json (hunk 1 above)
Comment (emlaver) [reply package.json:41]: Sounds good, I'll close this PR.

**Pull request closed without merging**